connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala

```diff
@@ -78,7 +78,7 @@ class SparkConnectClientSuite
   }
 
   test("Test connection") {
-    val testPort = 16000
+    val testPort = 16001
```

Comment:
Why did we change this?

Reply:
Two tests uses 16000 and cause some flakiness when running in parallel.